### PR TITLE
Fixing broken info box & making examples clearer

### PR DESCRIPTION
**Pull request by chrisschultzssw**
> What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

Email from Adam - "Markdown issue - Do you standardize AD group names? | SSW.Rules "

> What was changed?

- Fixed broken info box
- Changed examples to be clearer
- Removed a no longer relevant section

> Did you do pair or mob programming (list names)?

No.

### CODE REVIEW

File: rules/do-you-standardise-ad-group-names/rule.md [hunk 1]
```diff
@@ -26,21 +26,20 @@ The use of standardized group names is a simple yet crucial step towards easier
 For example, this is a list of AD groups associated with products:
 
 ::: greybox
- SSWSugarLearningEvents\
- SSWCodeAuditorAlerts\
- SSWLinkAuditorDevs
+ SSWSugarLearningAlerts\
+ Alerts CodeAuditor\
```
Comment (kikibianc) [inline rules/do-you-standardise-ad-group-names/rule.md:30]: Alerts CodeAuditor - this seems wrong?

**Review by kikibianc — CHANGES_REQUESTED** [review]
Check the comments

File: rules/do-you-standardise-ad-group-names/rule.md (hunk 1 above)
Comment (chrisschultzssw) [reply rules/do-you-standardise-ad-group-names/rule.md:30]: it is wrong, it's a bad example XD

**Review by kikibianc — APPROVED** [review]
Sweet

**Review by chrisschultzssw — COMMENTED** [review]
🚢

**Pull request merged**